components/feature/downloads/src/main/java/mozilla/components/feature/downloads/AbstractFetchDownloadService.kt

```diff
@@ -156,51 +167,15 @@ abstract class AbstractFetchDownloadService(
             mimeType = download.contentType ?: "*/*",
             path = file.absolutePath,
             length = download.contentLength ?: file.length(),
-            showNotification = true,
+            // Only show notifications if our channel is blocked
+            showNotification = !DownloadNotification.isChannelEnabled(context),
```

Comment:
This change I do not understand, why do we only want to show a notification if the channel is blocked? 🤔

Reply:
We try to show a custom notification and fall back to the system if our custom one is blocked.